src/codeu/chat/client/ClientUser.java

```diff
@@ -88,20 +94,24 @@ public void showCurrent() {
     printUser(current);
   }
 
-  public void addUser(String name) {
+  //change the type to a boolean, so we can check if the user was created or not in the GUI
```

Comment:
Great documentation!
You probably don't need to include "change the type to a boolean" however. Documentation is meant to help other readers of the code know what it does, so what it used to do is not relevant here.